src/CMakeLists.txt

```diff
@@ -1,19 +1,41 @@
 
 cmake_minimum_required(VERSION 3.1)
 
-project(Iaito VERSION 1.0.0)
+project(iaito VERSION 1.0.0)
 
-set(CMAKE_CXX_STANDARD 11)
+set(QT_MIN_VERSION "5.6.0")
 
-list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
+include(CheckCXXCompilerFlag)
+include(FeatureSummary)
+include(GenerateExportHeader)
+
+find_package(Qt5 ${QT_MIN_VERSION} NO_MODULE REQUIRED
+	WebEngine
+	WebEngineWidgets
+)
+
+find_package(Radare2 1.4.0 REQUIRED)
```

Comment:
FindRadare2.cmake (written by me) does not check the version, so this option will be ignored.

It might be helpful to add that functionality to the find module though, so people can't compile against any old versions.